### PR TITLE
5372 backup level iam operations

### PR DESCRIPTION
**Pull request by amitchauhan1007**

### CODE REVIEW

**Comment by amitchauhan1007** [comment]
seems like some issue with the merge. I made changes only in two files "table_admin.h" & "table_admin.cc"

File: google/cloud/bigtable/table_admin.h [hunk 1]
```diff
@@ -1534,6 +1611,13 @@ class TableAdmin {
     return instance_name() + "/clusters/" + cluster_id;
   }
 
+  /// Return the fully qualified name of a Cluster.
```
Comment (manish-qlogic) [inline google/cloud/bigtable/table_admin.h:1623]: cluster?

I think it should be `backupName`.

File: google/cloud/bigtable/table_admin.h [hunk 2]
```diff
@@ -1389,7 +1389,29 @@ class TableAdmin {
...
+   * @par Idempotency
+   * This operation is read-only and therefore it is always idempotent.
+   *
+   * @par Thread-safety
+   * Two threads concurrently calling this member function on the same instance
+   * of this class are **not** guaranteed to work. Consider copying the object
+   * and using different copies in each thread.
+   *
+   * @par Example
+   * @snippet table_admin_iam_policy_snippets.cc get iam policy
```
Comment (manish-qlogic) [inline google/cloud/bigtable/table_admin.h:1412]: this `get iam policy` snippet is not for backup, so either we write a snippet for `get iam policy bckup` or at least create a todo.

File: google/cloud/bigtable/table_admin.h [hunk 3]
```diff
@@ -1443,6 +1465,36 @@ class TableAdmin {
...
+   * @par Idempotency
+   * This operation is always treated as non-idempotent.
+   *
+   * @par Thread-safety
+   * Two threads concurrently calling this member function on the same instance
+   * of this class are **not** guaranteed to work. Consider copying the object
+   * and using different copies in each thread.
+   *
+   * @par Example
+   * @snippet table_admin_iam_policy_snippets.cc set iam policy
```
Comment (manish-qlogic) [inline google/cloud/bigtable/table_admin.h:1497]: same goes with this as well.

File: google/cloud/bigtable/table_admin.cc [hunk 4]
```diff
@@ -922,6 +922,31 @@ StatusOr<google::iam::v1::Policy> TableAdmin::GetIamPolicy(
   return proto;
 }
 
+StatusOr<google::iam::v1::Policy> TableAdmin::GetIamPolicy(
```
Comment (manish-qlogic) [inline google/cloud/bigtable/table_admin.cc:925]: we should write unit tests in `table_admin_test.cc`.

we should also write integration tests in `admin_iam_policy_integration_test.cc`, it can be a todo for now and can go in separate PR.

**Review by manish-qlogic — CHANGES_REQUESTED** [review]
